src/citrine/resources/material_spec.py

```diff
@@ -9,10 +9,11 @@
 from citrine._serialization.properties import List as PropertyList
 from citrine._serialization.properties import Optional as PropertyOptional
 from citrine._serialization.properties import String, LinkOrElse, Mapping, Object
-from citrine.resources.data_concepts import DataConcepts, CITRINE_SCOPE
+from citrine.resources.data_concepts import DataConcepts, CITRINE_SCOPE, _make_link_by_uid
```

Comment:
I think we can pretty consistently scrub the need to import `CITRINE_SCOPE`.

Reply:
It's still used in filter_by_template, so it stays for now (but that method is deprecated and will be removed shortly).

If the import wasn't used then flake8 would catch that

Reply:
But `filter_by_template` just calls `list_by_template`.  `scope: str = None` would just function to piggyback on `list_by_template`'s default.

Reply:
good point